WEB-GRIMM/README.md

```diff
@@ -649,6 +651,8 @@ Das Beispiel ist im Ordner `extra/fancy-input` zu finden.
   - sind immer dynamisch
   - appenden mit ``$name[] = "Franziska";``
   - es gibt auch assoziative Array, also mit benannten Indizes:
+- Funktionen unterstützen Standard-Parameter wie in C
```

Comment:
Seit wann gibt es in C Standard-Parameter?

Reply:
```suggestion
- Funktionen unterstützen Standard-Parameter wie in C++
```

Ups!